examples/application-headers.yml

```diff
@@ -1,15 +1,15 @@
-asyncapi: '2.5.0'
+asyncapi: 3.0.0
 info:
   title: Application Headers example
-  version: '1.0.0'
-  description: A cut of the Streetlights API to test application header changes supporting #112
+  version: 1.0.0
+  description: A cut of the Streetlights API to test application header changes supporting
```

Comment:
@jonaslagoni `#112` is missing here, otherwise the meaning is not correct.

Reply:
Hmm, it's a bug with the converter 🤔 

Mind adding a bug issue?